src/frontend/components/DashboardTimedTextManager/DashboardTimedTextManager.tsx

```diff
@@ -31,15 +31,21 @@ export class DashboardTimedTextManager extends React.Component<
   render() {
     const { message, mode, tracks } = this.props;
 
+    const existingLanguages: string[] = [];
+
     return (
       <DashboardTimedTextManagerStyled>
         <DashboardInternalHeading>
           <FormattedMessage {...message} />
         </DashboardInternalHeading>
-        {tracks.map(track => (
-          <TimedTextListItemConnected key={track.id} track={track} />
-        ))}
-        <TimedTextCreationFormConnected mode={mode} />
+        {tracks.map(track => {
+          existingLanguages.push(track.language);
```

Comment:
I think we're better off just plucking the languages in `existingLanguages` definition.

Reply:
This is what I made first and then I change for this solution because it avoids to call `map` twice.

Reply:
IMO it's way clearer to call map twice than to do some array mutation from inside a map iterator 😅

Reply:
Done